commons/commons-bio/src/main/kotlin/ebi/ac/uk/api/dto/SubmissionDto.kt

```diff
@@ -0,0 +1,10 @@
+package ebi.ac.uk.api.dto
+
+data class SubmissionDto (
+    var accno: String,
```

Comment:
I think you should use `val` for these properties. Also, please remove the space before the parenthesis.